api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -14,16 +14,19 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import javax.inject.Provider;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I never use this class before. Can we use javax.annotation.Nonnull?

Reply:
good catch, thanks

Reply:
Actually taking a closer look, this is our currently most popular option.  We have instances of all of these:
* org.jetbrains.annotations.NotNull
* javax.annotation.Nonnull
* javax.validation.constraints.NotNull

Maybe a question to ask the team for preference?